code-scanning/properties/clj-watson.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "clj-watson",
+    "creator": "Matheus Bernardes",
```

Comment:
`creator` can be some organisation or community. For now, individual names is not preferred

Reply:
Removed `creator` so that `GitHub` becomes the creator. @github/tpe will be the POC.